src/index.html

```diff
@@ -6,8 +6,97 @@
   </head>
-  <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+  <body class="page">
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__top">
+          <div class="header__hamburger"></div>
+          <div class="header__logo"></div>
+          <a href="#" class="header__phone"></a>
+        </div>
+        <h1 class="header__title">Miami<br> Condo Kings</h1>
+      </div>
+      <nav class="nav">
+        <div class="nav__icon-close"></div>
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#" class="nav__link">HOME</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">GALLERY</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">CONSULTATION</a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">TESTIMONIALS</a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">CONTACTS</a></li>
+        </ul>
+        <div class="nav__phone-number">+1 234 555-55-55</div>
+        <a href="#" class=" nav__call-order">CALL TO ORDER</a>
+      </nav>
+    </header>
+    <main>
+      <section class="page__container about">
+        <h2 class="about__title">Miami best<br> Real estate <span class="about__title_colored">here</span></h2>
+        <article class="about__article">
+          <h3 class="about__h3">Over 50 years experience in love</h3>
+          <p class="about__paragraph">
+            Mazel and VA Tsukkerman in his "Analysis of musical works."
+            Pointillism, which originated in the music of the early twentieth
+            century, microforms, found a distant historical.
+            <br><br>
+            The market leader in real estate since 2003. Awards Real Estate
+            Company, 2010, 2011, 2013. More than 200 satisfied customers premium
+            segment. We focus only on new buildings and best technologies. We
+            know this area better than other specialists. Working directly with
+            developers - so have more useful information about the profitable
+            deals.
+            <br><br><br>
+            <span class="about__paragraph_bold">We will never be late! Guaranteed secure transaction - 100%</span>
+          </p>
+        </article>
+      </section>
+      <section class="gallery page__container">
+        <h2 class="gallery__title">Gallery</h2>
+        <div class="crown gallery__crown"></div>
+        <div class="gallery__wrapper-1">
+          <img
+            src="./images/photos/1.jpg"
+            alt="photo-1"
+            class="gallery__photo"
+          >
+          <img
+            src="./images/photos/2.jpg"
+            alt="photo-2"
+            class="gallery__photo"
+          >
+        </div>
+        <div class="gallery__desciption">Marina Palms / North Miami Beach, FL 33162</div>
+        <div class="gallery__wrapper-2">
```

Comment:
```suggestion
        <div class="gallery__wrapper gallery__wrapper--1">
          <img
            src="./images/photos/1.jpg"
            alt="photo-1"
            class="gallery__photo"
          >
          <img
            src="./images/photos/2.jpg"
            alt="photo-2"
            class="gallery__photo"
          >
        </div>
        <div class="gallery__desciption">Marina Palms / North Miami Beach, FL 33162</div>
        <div class="gallery__wrapper">
```
Let's make one common class `gallery__wrapper` in which there will be all common styles for all your gallery-wrappers, but add modifier `--1` and `--3` for the first and for the last.
For this modifiers you should only write 
```
grid-template-columns: 1fr 2fr;
grid-template-columns: 2fr 1fr;
```

Also, use `--` instead of `-` for modifiers